pkg/apis/monitoring/v1/alertmanager_types.go

```diff
@@ -343,6 +345,37 @@ type AlertmanagerWebSpec struct {
 	Timeout *uint32 `json:"timeout,omitempty"`
 }
 
+// GlobalSMTPConfig configures smtp parameters that are valid in all other configuration contexts.
+// See https://prometheus.io/docs/alerting/latest/configuration/#configuration-file
+type GlobalSMTPConfig struct {
+	// The default SMTP From header field.
+	SMTPFrom string `json:"smtpFrom,omitempty"`
```

Comment:
To follow the [k8s recommendations](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required) could you please change all fields from values to pointers?

```suggestion
	// +optional
	SMTPFrom *string `json:"smtpFrom,omitempty"`
```

Reply:
👌